salt/modules/chocolatey.py

```diff
@@ -428,6 +429,11 @@ def install(name,
         no_progress
             Do not show download progress percentages. Defaults to False.
 
+        execution_timeout (str):
+        Chocolatey execution timeout value you want to pass to the installation process. Default is None.
+
+            .. versionadded:: 2017.7.3
```

Comment:
The version tag here should be `Oxygen` instead of `2017.7.3` as this is a new addition going into the `develop` branch.

Reply:
Done.